app-misc/vcontrold/vcontrold-0.98.8.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+man +vclient vsim"
+
+RDEPEND="dev-libs/libxml2"
```

Comment:
Imply with what slots it will work.

Reply:
There's only one slot of dev-libs/libxml2 available? Sorry, it has been quite some time since I committed an ebuild the last time … should it be RDEPEND="dev-libs/libxml2:2"?

Reply:
Or better RDEPEND="dev-libs/libxml2:="?

Reply:
`:=` is used to prevent breakage after librarys SONAME change, it is not needed here. Slot is used to help package manager a bit and to prevent breakages. This is good, thanks!